src/commands/edge_app.rs

```diff
@@ -525,6 +530,7 @@ impl EdgeAppCommand {
 
         fs::write(edge_app_dir.join(MOCK_DATA_FILENAME), mock_data_yaml)?;
 
+        println!("Generated mock data for Edge App Emulator.");
```

Comment:
you have this other message in this form:
>println!("Mock data for Edge App Emulator already exists.");

this one starts with generated. Let's be consistent

Reply:
I'll make it "Mock data for Edge App Emulator was generated."

Should it be with capital letters "Edge App Emulator" or lowercase "edge app emulator"?

Reply:
I think capital letters are OK. WE always capitalize Edge Apps at least.

Reply:
Ok, I've seen it as `Edge app/s` in couple places, I'll update it when i see it in lowercase.